base/essentials.jl

```diff
@@ -337,3 +337,16 @@ function vector_any(xs::ANY...)
 end
 
 isempty(itr) = done(itr, start(itr))
+
+"""
+    invokelatest(f, args...)
+
+Calls `f(args...)`, but guarantees that the most recent method of `f`
+will be executed.   This is useful in specialized circumstances,
+especially `cfunction` callbacks that may extract a Julia `Function`
```

Comment:
Doesn't need to mention cfunction anymore, it's not relevant there anymore. But perhaps should add a mention of long-running event loops?